api/validator/userValidator.js

```diff
@@ -53,9 +53,7 @@ module.exports = {
   validateUserProfileUpdate(req, res, next) {
     const validator = new Validator(req.body, {
       firstName: 'required|alpha',
-      lastName: 'required|alpha',
-      countryName: 'required|alpha',
-      cityName: 'required|alpha'
+      lastName: 'required|alpha'
     });
```

Comment:
for the location endpoint getting the id is there a validation that validates the country name and city name???

Reply:
It is sent from the autocomplete API to the front end...so the front end just returns the right city and country. But we can still have validation on the backend.